demo/api/src/app.module.ts

```diff
@@ -159,7 +159,15 @@ export class AppModule {
                 PredefinedPageModule,
                 CronJobsModule,
                 ProductsModule,
-                AccessLogModule,
+                AccessLogModule.forRoot({
+                    shouldLogRequest: ({ user }) => {
+                        // Ignore system user
+                        if (user === true) {
```

Comment:
I'm not sure if this is the way to go. Suppose the system user gets compromised, we loose all possibility to track potential misuse. I think this is only ok, if the system user is limited to internal networks.

Reply:
I agree with you 100%, but how do you currently want to find out whether the request is coming from an internal network?
And each application can decide for itself whether this check is made on the system user to skip the log.

Reply:
@dkarnutsch are you okay with this?